teapots/common/ndk_helper/JNIHelper.h

```diff
@@ -151,23 +235,31 @@ class JNIHelper {
   std::string GetExternalFilesDir();
 
   /*
+   * Retrieve string resource with a given name
+   * arguments:
+   *  in: resourceName, name of string resource to retrieve
+   * return: string resource value, returns "" when there is no string resource with given name
+   */
+  std::string GetStringResource(const std::string& resourceName);
+
+  /*
    * Audio helper
    * Retrieves native audio buffer size which is required to achieve low latency
-   *audio
+   * audio
    *
    * return: Native audio buffer size which is a hint to achieve low latency
-   *audio
+   * audio
    * If the API is not supported (API level < 17), it returns 0
    */
   int32_t GetNativeAudioBufferSize();
```

Comment:
same as before.  and this is not purpose for this code change, so please waive it :-)